### PR TITLE
🤖 backported "Split SummarizeContent from SummarizeSidebar"

### PR DESCRIPTION
**Pull request by github-automation-metabase**
#44539
  > [!IMPORTANT]
  > Manual conflict resolution is required.
  Checkout the branch and run `./backport.sh` script. Force push your changes after cherry-picking.

### CODE REVIEW

**Review by npretto — APPROVED** [review]

File: frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  SectionTitle,
+} from "../SummarizeSidebar.styled";
+
+import { STAGE_INDEX } from "./use-summarize-query";
+
+export type SummarizeContentProps = {
+  query: Lib.Query;
+  aggregations: Lib.AggregationClause[];
+  hasAggregations: boolean;
+  onAddAggregation: (aggregations: Lib.Aggregable) => void;
```
Comment (nemanjaglumac) [inline frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx:20]: Why is this different than the original PR where it's `onAddAggregations`?

Comment (oisincoveney) [reply frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx:20]: I had the same question. It looks like the `onAddAggregations` functionality was never added, and I was getting quite a few errors down the component tree.

Comment (oisincoveney) [reply frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx:20]: Let me know how to proceed if it's an issue here

Comment (oisincoveney) [reply frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx:20]: Based on our conversation, [**#43220 Previous period comparison shortcut in notebook mode**](https://github.com/metabase/metabase/pull/43220) not being backported will let us backport without much concern :crossed\_fingers:

**Pull request merged**